### PR TITLE
Fix and further improve status bar & nav bar

### LINKED ISSUE #297: Login screen bugged into status bar
It seems one of the last commits led to a bug.  
Currently the login screen is bugged into the status bar, so the Logs button and the logo disappears into the top.

### PR DESCRIPTION
**Pull request by cyb3rko**
I've replaced the transluscent status bar with a manually colored one.  
This prevents bugging of UI elements to the top and allows better control over the wanted color.

Furthermore the v21 style file could be deleted as the minSdk is currently already at 23.

One thing that still  seems a bit off are the different colors of the action bar and the navigation drawer because of the changed colorPrimary in #292.

What do you think?

Closes #297

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (jmattheis) [inline app/src/main/AndroidManifest.xml:40]: > One thing that still seems a bit off are the different colors of the action bar and the navigation drawer because of the changed colorPrimary in https://github.com/gotify/android/pull/292.

I'd be okay with reverting this back to the old one. Tho having a different color for the top bar and the navigation drawer isn't that bad, because it adds a little contrast, what's your opinion on this?

**Review by jmattheis — APPROVED** [review]
Looks great, thanks! I'll merge this, after we decided the color thingy in my subcomment.

File: app/src/main/AndroidManifest.xml
Comment (cyb3rko) [reply app/src/main/AndroidManifest.xml:40]: I'll leave that up to you.  
We can leave it as is.

Comment (jmattheis) [reply app/src/main/AndroidManifest.xml:40]: oke, then let's leave it as it. Less effort (:.

**Comment by jmattheis** [comment]
FYI: The build just broke because a dependency cannot be resolved anymore. Not the fault of this PR. I've used a different logging library in another project. I'll probably replace hyperlog with the other one.

**Comment by cyb3rko** [comment]
Interesting, how did it break?
We didn't update anything nor does it use a backend?

**Comment by cyb3rko** [comment]
@jmattheis It seems to work again, right?

**Comment by jmattheis** [comment]
yeah

**Pull request merged**
